AdyenComponents/BACS Direct Debit/BACSDirectDebitComponent.swift

```diff
@@ -22,9 +22,7 @@ public final class BACSDirectDebitComponent: PaymentComponent, PresentableCompon
     // MARK: - PresentableComponent
 
     /// :nodoc:
-    public var viewController: UIViewController {
-        SecuredViewController(child: inputFormViewController, style: configuration.style)
-    }
+    public let viewController: UIViewController
```

Comment:
😅

BTW, this could be just :
```suggestion
    public lazy var viewController: UIViewController = SecuredViewController(child: inputFormViewController,  style: configuration.style)
```